internal/envoy/config.go

```diff
@@ -36,11 +36,11 @@ type ConfigWriter struct {
 	AdminPort int
 
 	// StatsAddress is the address that the /stats path will listen on.
```

Comment:
contour metrics path is `/metrics`. Is the comment outdated or am I missing something? :)

Reply:
Oh, this is the envoy part. This is now `/stats/prometheus`

Reply:
yeah, its a bit confusing, this is the address of the listener we push to envoy for it to serve it's stats on.

The comment could probably be improved with ^^ that text.

Reply:
:+1: